### ISSUE TITLE
Signature and Certificate formats and content

### ISSUE THREAD

**Issue by kusala9** [iho-ohi/S-164-Sub-Group#6]
## Introduction / Background
S-100WG7 agreed paper S100WG7-6.7, Identifiers and Procedures which (amongst other things) defined content to be embedded in all X.509 certificates used by data protection scheme participants implementing Part 15 of S-100 edition 5.0.0. These were defined as:

## X.509 Fields (mandatory)
•	C (Country) = ISO Country Code of state making request
•	ST (State or Province) = A code reflecting the role of the subject
•	O (Organisation) = member state organisation name (text)
•	CN (Common Name) =  IHO data producer code integer and alpha code (as part of a colon separated MRN), e.g. urn:mrn:iho:aa:1810 or urn:mrn:iho:GB:540.

NIPWG VTC 19th June 2023 agreed the following format for CN

**urn:mrn:iho:org:<alphanumeric code>:<numeric code>**

which will be passed back to S-100WG and IHO secretariat for implementation.

Roles need defining (another issue to deal with this). Test data can be defined once these are all agreed.

**Comment by kusala9** [comment]
Once test data is complete and no issues identified with the naming scheme and how it is used this issue can be closed.

**Comment by kusala9** [comment]
test keys/certs added to repository under https://github.com/iho-ohi/S-164-Sub-Group/commit/bdc5f6919cf1631df6f6a5d94a7c84e0884c06c9 - these have the fields defined as per above.

**Comment by kusala9** [comment]
close this - superceded by existing draft of S-98 Annex C (specification, as agreed by IHO) and existing datasets. This is now in S-98 Annex C C-12.1.1
